src/main/java/com/codeborne/selenide/logevents/SimpleReport.java

```diff
@@ -208,6 +218,36 @@ private static void checkThatSlf4jIsConfigured() {
   }
 
   @Desugar
-  private record LogEventWithNestingLevel(int nestingLevel, LogEvent event) {
+  private record LogEventWithNestingLevel(int nestingLevel, SimpleLogEvent event) {
+
+    @Desugar
+    private record SimpleLogEvent(
+      String getElement,
+      String getSubject,
+      EventStatus getStatus,
+      long getDuration,
+      long getStartTime,
+      long getEndTime,
+      Throwable getError
+    ) implements LogEvent {
+    }
+  }
+
+  private static String escape(String text) {
+    var builder = new StringBuilder((int) (text.length() * 1.5));
+    for (int i = 0; i < text.length(); i++) {
+      var symbol = text.charAt(i);
+
+      switch (symbol) {
+        case '\t' -> builder.append("\\t");
+        case '\r' -> builder.append("\\r");
+        case '\n' -> builder.append("\\n");
+        case '\f' -> builder.append("\\f");
+        case '\b' -> builder.append("\\b");
+        default -> builder.append(symbol);
```

Comment:
@Au6ojlut Maybe we should also escape all invisible characters below 32 (space)?

Reply:
Agree, I'll fix it

Reply:
Done